src/generator/www/js/main.js

```diff
@@ -3,6 +3,7 @@ $(document).ready(function() {
   let $email =$("#email");
   let $name  =$("#name"); 
   let $url  =$("#url");
+  let $theme =$("#theme");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'let' statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2738101723/issues/source?bid=3233857&fileBranchId=3347760#l6)